dist/tools/kconfiglib/tests/test.sh

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2020 HAW Hamburg
+#               2020 Kaspar Schleiser <kaspar@schleiser.de>
+#               2020 Inria
+#               2020 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+# This script performs some tests on the genconfig.py script. It uses as main
+# Kconfig tree file 'Kconfig.test'. It runs the script under different
+# conditions by injecting different configuration files and checking the return
+# values.
+
+KCONFIGLIB_DIR="$(cd $(dirname $0)/..; pwd)"
```

Comment:
Is this identical to this
```suggestion
KCONFIGLIB_DIR="$(dirname "$(readlink -f "$0")")"
```
?